6.bulk_Morphology_Elastic_Network/2.test_model/notebooks/1.test_regression_multi_output.ipynb

```diff
@@ -1,428 +1,512 @@
 {
```

Comment:
<p>What does the "NSU" indicate? Consider describing this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Normalized Standard Units I added documentation to clarify. Good point!</p>